contracts/test/integration/sentinel/FuseWithdrawalGuard.t.sol.disabled

```diff
@@ -54,7 +54,9 @@ contract FuseWithdrawalGuardIntegrationTest is DSTest, StdLib {
         sentinel.knight(address(guard));
     }
 
-    function testGuardCanProtec() public {
+    // TODO: update Fuse withdrawal guard PCV_GUARDIAN address,
```

Comment:
I think the guardian should be deployed locally in this integration test and then it's address be injected into the `FuseWithdrawalGuard` through it's constructor, otherwise you can't test the guard without deploying something to mainnet first

Reply:
good idea